src/cdk/bootstrap.ts

```diff
@@ -275,7 +318,7 @@ export class MiraBootstrap {
 
     for (const key of Object.keys(args)) {
       if (args[key]?.includes && args[key].includes(' ')) {
-        const q = process.platform !== 'win32' ? '\'' : '"'
+        const q = process.platform !== 'win32' ? "'" : '"'
```

Comment:
Similar to above, wondering what this solves?  Was there an error?

Reply:
This is just a linter fix, strip the useless `\`